tsl/src/bgw_policy/compression_api.c

```diff
@@ -297,17 +331,38 @@ policy_compression_add(PG_FUNCTION_ARGS)
 	PG_RETURN_INT32(job_id);
 }
 
+/* remove compression policy from ht or cagg */
 Datum
 policy_compression_remove(PG_FUNCTION_ARGS)
 {
-	Oid hypertable_oid = PG_GETARG_OID(0);
+	Oid user_oid = PG_GETARG_OID(0);
```

Comment:
I find this variable name very confusing. This doesn't seem to be an OID of a user. Could we come up with a better name?

Reply:
Agreed, it's also used in `policy_compression_add()`. Maybe `rel_oid` or `user_rel_oid`?

Reply:
renamed as user_rel_oid. I used user to indicate that this is user supplied oid.  I do see  that it might be interpreted as user id.

Reply:
I don't see the rename. Can you rename this for `policy_compression_add()` as well?

Reply:
was not pushed.